libtcmu_config.c

```diff
@@ -404,54 +404,63 @@ static void *dyn_config_start(void *arg)
 	struct tcmu_config *cfg = arg;
 	int monitor, wd, len;
 	char buf[BUF_LEN];
+	char cfg_dir[PATH_MAX];
+	char *p;
 
 	monitor = inotify_init();
 	if (monitor == -1) {
 		tcmu_err("Failed to init inotify %m\n");
 		return NULL;
 	}
 
-	wd = inotify_add_watch(monitor, cfg->path, IN_ALL_EVENTS);
+	snprintf(cfg_dir, PATH_MAX, cfg->path);
+	p = strrchr(cfg_dir, '/');
```

Comment:
Before it will allowed us to use /AA/BB, but now only /AA/BB/ is allowed.
Is this okay ?

Reply:
BTW - this is just conf file's dir path, not the log_dir path.

Reply:
yeah.